src/components/index/sections/Jumbotron.tsx

```diff
@@ -36,8 +40,9 @@ const useStyles = makeStyles((theme) =>
     podkrepiButton: {
       color: theme.palette.common.white,
       borderColor: theme.palette.common.white,
-      padding: theme.spacing(2, 4),
+      padding: theme.spacing(1.5, 5),
       fontWeight: 600,
+      fontSize: theme.spacing(2),
```

Comment:
For font sizes we can use `fontSize: theme.typography.pxToRem(15)` which gives us similar result as spacing with more responsive